Flutter/pdf-viewer/select-deselect-annotations.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Select and Deselect Annotations in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about select and deselect annotations in PDF documents using the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Select and Deselect Annotations in Flutter PDF Viewer widget (Syncfusion)
+
+This section will go through the various functions available in the `SfPdfViewer` for selecting and deselecting annotations in a PDF document.
+
+## Select an annotation
+
+You can select an annotation by simply tapping on the annotation using touch or mouse. When the annotation is selected, the selection border (selector) appears, indicating that the annotation is selected. 
+
+### Select an annotation programmatically
+
+You can select an annotation programmatically by providing the annotation instance as the parameter to the `selectAnnotation` method of PdfViewerController. The annotation instance can be found in the `getAnnotations` method of the PdfViewerController. The following example explains how to select the first annotation in the annotation collection.
+
+{% tabs %}
+{% highlight dart hl_lines="8" %}
+
+void selectFirstAnnotation() {
+  // Get the list of annotations in the PDF document.
+  List<Annotation> annotations = _pdfViewerController.getAnnotations();
+  if (annotations.isNotEmpty) {
+    // Get the first annotation from the PDF document.
+    Annotation firstAnnotation = annotations.first;
+    // Select the first annotation in the PDF document.
+    _pdfViewerController.selectAnnotation(firstAnnotation);
+  }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Annotation selected callback
+
+The callback provided to `onAnnotationSelected` property is triggered when an annotation is selected interactively or programmatically. The selected annotation instance can be obtained from the callback details. The following code snippet explains how to use this callback.
```

Comment:
The callback provided to the `onAnnotationSelected` property is triggered when an annotation is selected interactively or programmatically. The selected annotation instance can be obtained from the callback details. The following code sample explains how to use this callback.